src/sentry/static/sentry/app/actionCreators/projects.jsx

```diff
@@ -60,20 +76,27 @@ const _debouncedLoadStats = debounce((api, projectSet, params) => {
     return;
   }
 
-  let idQueryParams = projects.map(project => `id:${project}`).join(' ');
-  let endpoint = `/organizations/${params.orgId}/projects/`;
-
-  api.request(endpoint, {
-    query: {
-      statsPeriod: '24h',
-      query: idQueryParams,
-    },
-    success: data => {
-      ProjectActions.loadStatsForProjectSuccess(data);
-    },
-    error: data => {},
+  const queries = [
+    ...Array(Math.ceil(projects.length / MAX_PROJECTS_TO_FETCH)),
+  ].map((val, index) => {
+    const start = index * MAX_PROJECTS_TO_FETCH;
+    return _queryForStats(
+      api,
+      projects.slice(start, start + MAX_PROJECTS_TO_FETCH),
```

Comment:
Won't you fetch some projects twice with this?

* index = 0 gets 0-10
* index = 1 gets 10-20
* index = 2 gets 20 - 30

Should the max boundary be `start + MAX_PROJECTS_TO_FETCH - 1`?

Reply:
The end boundary doesn't get included `[0, 1, 2, 3].slice(0, 2)` will return `[0, 1]`

Reply:
Ok. I missed the `.slice()` call my bad.

Reply:
chunk (from lodash) could simplify this code quite a bit